Earthfile

```diff
@@ -9,3 +11,26 @@ proto:
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf build
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf generate
   SAVE ARTIFACT gen/proto/go/translate/v1 AS LOCAL pkg/server/translate/v1
+
+lint-go:
+  FROM golang:$go_version-alpine
+  RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v$golangci_lint_version
+  RUN apk add build-base
+  WORKDIR /translate
+  COPY --dir pkg .
+  COPY .golangci.yml .
+  COPY go.mod go.sum .
+  RUN go mod download
```

Comment:
Don't like this, as firstly: redownloads every time, if something is changes as we don't save the artifact, secondly: not really the thing that target should be doing.
My suggestion, another target that generates those files, go mod init, etc. and then saves as local artifact